### DISCUSSION TITLE
macos app with gitlab-ci

### DISCUSSION THREAD

**Discussion by brusand** [pyinstaller/pyinstaller#8411] [Help]
doas anyone had succed in building a mac app with gitlab-ci pipeline ? if yes how, wich image do you use ?
thx
Bruno

**Comment by bwoodsend** [comment]
You're not going to get a meaningful answer without explaining where it's going wrong.

**Comment by brusand** [comment]
i have no idea to do this, i think i need an macos image with  python 3.8 and pyinstaller ? i didn t find any macos image for gitlab-ci.
is a python image is suffisant to build both windows ans mac native app ? so my question is is there someone had already use gitlab-ci to build and package windows and mac app
